lib/tag_editor.dart

```diff
@@ -17,6 +17,7 @@ class TagEditor extends StatefulWidget {
     this.icon,
     this.enabled = true,
     // TextField's Props
+    this.textStyle = const TextStyle(),
```

Comment:
I believe that `TextField` can handle `null` for style so I don't think you need the default value here.
```suggestion
    this.textStyle,
```
should be ok